h/static/styles/components/_annotation-card.scss

```diff
@@ -8,13 +8,16 @@
 
 .annotation-card {
   @include typography.font-normal;
-  border: 1px solid color.$grey-2;
-  list-style: none;
-  padding: 15px;
-  margin-bottom: 10px;
-  box-shadow: 0px 1px 1px 0px rgba(0, 0, 0, 0.1);
-  background: white;
-  overflow-wrap: break-word;
+
+  & {
+    border: 1px solid color.$grey-2;
+    list-style: none;
+    padding: 15px;
+    margin-bottom: 10px;
+    box-shadow: 0px 1px 1px 0px rgba(0, 0, 0, 0.1);
+    background: white;
+    overflow-wrap: break-word;
+  }
```

Comment:
I think I don't get what is SASS warning about here. This is a simple class selector with a bunch of props 🤔 

Is it because of the presence of `@include typography.font-normal;`?

I should probably start by reading the link you added to the description 😅

Reply:
> Is it because of the presence of @include typography.font-normal;?

Yes. My impression is that the warning is produced by examining the AST after merging any mixins into the current node.